scripts/docker/Dockerfile.sycl

```diff
@@ -36,7 +36,7 @@ RUN CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSIO
 ENV PATH=${CMAKE_DIR}/bin:$PATH
 
 ENV SYCL_DIR=/opt/sycl
-RUN SYCL_VERSION=20210311 && \
+RUN SYCL_VERSION=20210621 && \
```

Comment:
This matches recent compiler releases reasonably well (the release was around 06/23).